concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1815,8 +1815,12 @@ jobs:
           file: paas-cf/concourse/tasks/smoke-tests-run.yml
 
         ensure:
-          task: remove-temp-user
-          file: paas-cf/concourse/tasks/delete_admin.yml
+          do:
+          - task: get-cf-cli-config
+            file: paas-cf/concourse/tasks/get-cf-cli-config.yml
+
```

Comment:
regarding multiple parsing of same thing, I thikn we could perhaps login once and then export `~/.cf/config.json` and just include that into tasks that need a logged-in CLI. But that sounds like a "support cleanup/optimize" task or something for our [improvements spreadsheet](https://docs.google.com/spreadsheets/d/1034dgtWbchumi-xJuTP2n4Iq8KpxF20Vrwgy3txmeBg/edit#gid=1419569599)

In this case, doing parsing of information as a separate task  adds a lot of repetetion into our already quite huge pipelines...